usecases/connect-autonomous-database/test/README

```diff
@@ -0,0 +1,50 @@
+Wallet Expiry Date
+-----------------------
+This wallet was downloaded on 2022-09-24 09:55:44.073237 UTC.
+The SSL certificates provided in this wallet will expire on 2027-09-23 03:01:11.295953 UTC.
+In order to avoid any service interruptions due to an expired SSL certificate, you must re-download the wallet before this date.
+
+Autonomous Database Tools and Resources
+---------------------------------------
+
+Database Actions
+----------------
+Load, explore, transform, model, and catalog your data. Use an SQL worksheet, build REST interfaces and low-code apps, manage users and connections, build and apply machine learning models. 
+Access Link:
+
+https://G357302A3DDDAD4-DEMODB.adb.us-phoenix-1.oraclecloudapps.com/ords/sql-developer
+
+Graph Studio
+------------
+Oracle Graph Studio lets you create scalable property graph databases. Graph Studio automates the creation of graph models and in-memory graphs from database tables. It includes notebooks and developer APIs that allow you to execute graph queries using PGQL (an SQL-like graph query language) and over 50 built-in graph algorithms. Graph Studio also offers dozens of visualization, including native graph visualization. 
+Access Link:
+
+https://G357302A3DDDAD4-DEMODB.adb.us-phoenix-1.oraclecloudapps.com/graphstudio/
```

Comment:
This doesn't seem like the correct URL.  Shouldn't a URL that's specific to the ADB instance be used (rather than a hard-coded URL)?

Reply:
Removed these files as these are generated by OCI ADB